src/main/java/ca/corefacility/bioinformatics/irida/model/project/Project.java

```diff
@@ -305,6 +305,7 @@ public void setSistrTypingUploads(AutomatedSISTRSetting sistrTypingUploads) {
 	 * AUTO - Run SISTR
 	 * AUTO_METADATA - Run and save results to metadata
 	 */
+	//TODO REMOVEME
```

Comment:
Why did you set this as a `TODO`?  Is this something that need to be handled in this merge request?

Reply:
I'm going to remove it in a later merge where I translate all the automated assemblies/SISTR into `AnalysisSubmissionTemplate`.  I just marked it here because I noticed it.  I changed it to be `@Deprecated` in 31a0bf827d